dired.py

```diff
@@ -1120,7 +1120,16 @@ def run(self, edit, project_folder=False):
                     subprocess.Popen(['sublime'] + items, cwd=self.path)
 
         def run_on_new_window():
-            sublime.active_window().run_command("dired", {"immediate": True, "project": True, "other_group": "left"})
+            settings = sublime.load_settings('dired.sublime-settings')
+            open_on_jump = settings.get('dired_open_on_jump')
```

Comment:
Default value should be set for fallback in case of missing settings file(s).
Settings could be missed because of failed package upgrade or user can remove them for [no] some reason.

Could you also squash? I mean it is one feature, would be nice to keep it in one commit (clean history make debug easier in general). If no, I’ll do it today later.